src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class ErrorBoundary extends React.Component<any, { hasError: boolean }> {
```

Comment:
Please remove unused Class. If you are intending to use it in the future then disregard this comment. :)